crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use axum::{
+    http::{HeaderMap, HeaderValue, Request, StatusCode},
+    middleware::Next,
+    response::{IntoResponse, Response},
+};
+
+use crate::error::{code, graphql_error};
+
+const VERSION_HEADER: &str = "X-Sui-RPC-Version";
+
+const RPC_VERSION_FULL: &str = env!("CARGO_PKG_VERSION");
+const RPC_VERSION_YEAR: &str = env!("CARGO_PKG_VERSION_MAJOR");
+const RPC_VERSION_MONTH: &str = env!("CARGO_PKG_VERSION_MINOR");
```

Comment:
I take it these are just mocks for now?

Reply:
The fact that it's "0.1.0" is a mock for now, yes.  We'll start using the actual year and month once we are ready to release.